### DISCUSSION TITLE
Integration with OMNET++

### DISCUSSION THREAD

**Discussion by ana-luisa00** [eclipse/mosaic#299] [Q&A]
While trying to integrate OMNET++ the script tells me that I do not have Python installed and to install it, however it is installed on my machine as you can see below

![Screenshot from 2023-03-15 15-43-20](https://user-images.githubusercontent.com/87331760/225363530-bf83acff-8799-4d25-8605-96cf2fee16a6.png)

Any idea of why this might be happening?

**Comment by schwepmo** [comment]
I have limited experience with using Omnet++ but from what I can figure is that the script is looking for `python` without the `3`. To check this you can type in `python`, which will probably lead to an error.
A simple solution would be to set an alias but there are different solutions, which you can find here https://askubuntu.com/questions/320996/how-to-make-python-program-command-execute-python-3
